lti_redirect/views.py

```diff
@@ -46,7 +51,19 @@ def get_restrucutured_data(launch_data):
     print(restructured_data)
     return restructured_data
 
-
+def login_user_from_lti(request, launch_data):
+    try:
+        first_name = launch_data['given_name']
+        last_name = launch_data['family_name']
+        email = launch_data['email']
+        username = launch_data['https://purl.imsglobal.org/spec/lti/claim/custom']['login_id']
+        user_obj = User.objects.get(username=username)
+    except User.DoesNotExist:
+        password = ''.join(random.sample(string.ascii_letters, settings.RANDOM_PASSWORD_DEFAULT_LENGTH))
+        user_obj = User.objects.create_user(username=username, email=email, password=password, first_name=first_name,
+                                            last_name=last_name)
+        
+    django.contrib.auth.login(request, user_obj)
```

Comment:
This code add the LTI user to the django `auth_user` table. Once the LTI launch happen, you can go to this table and see if the user is created  or not.